### ISSUE TITLE
Read replicas for ingest

### ISSUE THREAD

**Issue by matschaffer** [Safecast/safecastapi#347]
We should set up read replicas for ingest. Ideally these should be able to host their own copies or materialized views of the source db.

Once it's working here are some things we'd like to try hooking to it:

- [ ] [MS Office Power Query](https://support.office.com/en-us/article/Connect-to-a-PostgreSQL-database-Power-Query-bf941e52-066f-4911-a41f-2493c39e69e4)
- [ ] [QGIS](http://www.qgis.org/en/site/)

**Comment by matschaffer** [comment]
I have no idea what I'm doing but QGIS seems to be working as expected against the replica.

![screen shot 2017-01-26 at 15 14 25](https://cloud.githubusercontent.com/assets/690/22321688/26c39e3a-e3da-11e6-9dda-d8513f721d92.png)

**Comment by eitoball** [comment]
> I have no idea what I'm doing but QGIS seems to be working as expected against the replica.

It seems that there are string? like `0101000020E6100000D97C5C1B2AB251C06E5166834C484540` in location column. I guess that PostGIS correctly interprets data in that column because DMS just copies as is.

**Comment by matschaffer** [comment]
Yes, the DMS version didn't seem to work out for exactly that reason. Could be there's some way to trick it. 

Schema was also not completely replicated. Indexes seemed to get lost in the ETL.

So just doing regular replicas for now. Downside is you can't do the MySQL trick where you enable write on the slave as well. Guess it's safer that way though. If the slave tries to write to something that conflicts with the master you'd have to completely restart the replication.
